src/Microsoft.TemplateEngine.Cli/LocalizableStrings.resx

```diff
@@ -434,4 +434,36 @@
     <value>{0} {1}
     The default value '{1}' is not a valid value for {0}.</value>
   </data>
+  <data name="AddRefPostActionFailed" xml:space="preserve">
+    <value>Failed to add reference {0} to project file {1}</value>
+  </data>
+  <data name="AddRefPostActionMultipleProjFiles" xml:space="preserve">
+    <value>Found multiple possible project files in the same directory. Could not determine which to add a reference to. Files:</value>
+  </data>
+  <data name="AddRefPostActionNoProjFileFound" xml:space="preserve">
+    <value>Unable to add reference {0}. Could not find an appropriate project file.</value>
+  </data>
+  <data name="AddRefPostActionNoRefConfigured" xml:space="preserve">
+    <value>No references configured to be added to a project.</value>
+  </data>
+  <data name="AddRefPostActionAddProjectRef" xml:space="preserve">
+    <value>Adding a project reference. Running dotnet add {0} reference {1}</value>
+  </data>
+  <data name="AddRefPostActionSucceeded" xml:space="preserve">
+    <value>Successfully added
+    reference: {0}
+    to project file: {1}</value>
```

Comment:
Is there going to be any line ending issues here? Does it render the same on windows/mac?